lib/src/actions/entity-action-factory.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('EntityActionFactory', () => {
 
   it('#create should create an EntityAction with options', () => {
     const options: EntityActionOptions = {
-      correlationId: 'CRID42',
+      correlationId: 'CRUD42',
```

Comment:
Actually "CRID" is correct. It stands for "_C_o_r_relation_Id_".

Reply:
allright.